client/src/client.rs

```diff
@@ -103,7 +104,7 @@ impl Default for Config {
             root: PathBuf::from(env::var("HOME").unwrap_or_default()),
             target_outbound_peers: p2p::protocol::connmgr::TARGET_OUTBOUND_PEERS,
             max_inbound_peers: p2p::protocol::connmgr::MAX_INBOUND_PEERS,
-            services: ServiceFlags::NONE,
```

Comment:
The default should be `NONE`, because these are the advertised services of the nakamoto client. `NETWORK` is for full nodes. Instead, you should overwrite the default in the test.